### PR TITLE
Gas Mask Variety Take Two (Not Summoning Every Maintainer Edition)

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds three variants of the gas mask, replaces fire locker gas masks with a random pick between two of the new variants.

image

## Why It's Good For The Game
Gas masks are a go to mask, because they both hide your identity AND can be used as internals. This doesn't remove the old gas masks, but makes them much harder to find, and replaces the most common sources with one of three new variants that makes trade offs between each other, so there are less of the all powerful greytide masks everywhere. First this and then i'll make gas masks actually work like gas masks ok??

## Changelog
:cl:
add: Added three new gas mask variants; one which doesn't hide identities or block gases, one which doesn't hide eyes or block gases, and a final one which is purely aesthetic & can't be used as internals, nor blocks gases.
balance: fire locker gas masks are now chosen randomly between the first two new ones, as well as a normal gas mask.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
just gonna take this from the old one

That comment doesn't even make sense though, how does it make sense to make gas masks, the thing you are supposed to have when fighting fire, not be able to be connected to internals? I don't even get whats "good" about the mask, its a liable reason to be arrested by shitsec so any real advantage it provides is ???? This PR seems like pointless bloat tbh

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -11,6 +11,39 @@
...
+	gas_transfer_coefficient = 0.005 //not that this will ever be used but there you go
+	permeability_coefficient = 0.005
+	item_state = "gas_alt3"
+
+/obj/item/clothing/mask/gas/alt3
+	name = "gas mask"
+	desc = "A face-covering mask that cannot be connected to an air supply. Turns out to not be that great at blocking gasses."
+	icon_state = "gas_alt4"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 10, "rad" = 10, "fire" = 5, "acid" = 5)
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT
```
Comment (ShizCalev) [inline code/modules/clothing/masks/gasmask.dm]: i concur with the sentiments in the other thread honestly, having a gas mask that appears in an emergency fire closet that can't actually be used as internals is pretty shitty. If you want to conceal your identity, there's plenty of bandanas and crap laying around the station already that will do that for you.

**Comment by Shadowflame909** [comment]
It being a bunch of new sprite changes instead of a rebalance would be pretty great

**Comment by Ohagi-Chan** [comment]
"replaces the most common sources with one of three new variants that makes trade offs between each other"

Can we get a description of their differences in the pr description?
Blink and you might accidentally assume this is purely a sprite change.

**Comment by ShizCalev** [comment]
Edited your changelog to be a little more descriptive.

**Comment by MrPerson** [comment]
I'm inclined to agree with everyone else that just the sprite changes would have been fine and very welcome.

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -11,6 +11,39 @@
...
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEEARS|HIDEFACE|HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_NORMAL
+	gas_transfer_coefficient = 0.005 //not that this will ever be used but there you go
+	permeability_coefficient = 0.005
+	item_state = "gas_alt3"
+
+/obj/item/clothing/mask/gas/alt3
+	name = "gas mask"
+	desc = "A face-covering mask that cannot be connected to an air supply. Turns out to not be that great at blocking gasses."
```
Comment (Qustinnus) [inline code/modules/clothing/masks/gasmask.dm]: description does not match flags

**Comment by Michael-Ashfield** [comment]
Do the non-internals ones at least filter toxins from the air so you're not huffing hot plasma through a filtered gasmask while fighting a plasma fire?

**Comment by Dorsisdwarf** [comment]
I dont like the idea of putting a completely worthless fake gas mask on the station to trick people into dying.

**Comment by TheVekter** [comment]
Yeah, no, this reeks of ided and seems very disingenuous. I see no problem making these randomly spawn around the station, as they're good sprites (and the clear mask looks _much_ closer to a normal fireman's mask) but having them functionally different is just unnecessary.

**Comment by Tlaltecuhtli** [comment]
is it really that hard to just add different gas mask sprites without having to add """balance changes""""

**Comment by AWalton** [comment]
I don't see the point in having a gas mask which can't be used as internals, even if it isn't found in fire lockers and only in maint. 

There are a ton of different masks that people can wear to hide their identity instead. Also some helmets I believe don't hide identity like cardborg and laser tag helmet which should be rebalanced to do so as well. 

Many masks which shouldn't intuitively work as internals like the clown and mime mask do in fact, because why would anyone ever opt for a non-internals mask when they're needed so frequently and internals masks are so commonplace. 

And having a non-internals gas mask is just going to get people killed when they throw away their breath mask unknowingly and get caught in a gas flood or depressurised area.

**Comment by Iatots** [comment]
I WISH people cared about random unknowns wearing gasmasks and their work attire - ID.

**Comment by TheVekter** [comment]
> 
> 
> I WISH people cared about random unknowns wearing gasmasks and their work attire - ID.

Off-topic but I always ask if I'm working sec.

**Comment by Paxilmaniac** [comment]
fuck it i might as well make gas masks filter gasses like actual gas masks while im at it

**Pull request closed without merging**